common/test/unit/test_stats_service/test_stats_manager.py

```diff
@@ -124,9 +128,71 @@ def test_default_configs(self):
       "all": 4,
       "4xx": 2,
       "5xx": 1,
+      "latency": 0,
       "by_app": {
-        "guestbook": {"all": 2, "4xx": 1, "5xx": 0},
-        "ghostbook": {"all": 2, "4xx": 1, "5xx": 1}
+        "guestbook": {"all": 2, "4xx": 1, "5xx": 0, "latency": 0},
+        "ghostbook": {"all": 2, "4xx": 1, "5xx": 1, "latency": 0}
+      }
+    })
+
+
+class TestDefaultCumulativeSummarizers(unittest.TestCase):
```

Comment:
If you look to previous tests structure you can find that cumulative counters are tested by two test cases: `TestDefaultCumulativeCounters` which tests default configs and `TestCustomCumulativeCounters` which tests custom config with custom matcher.

At least two those classes should be extended to cover summarizers functionality.
In particular for test of custom counters you need to implement example of custom summarizer.